README.md

```diff
@@ -366,9 +347,10 @@ Also included is:
 
 ### Chisel Sub-Projects
 
-Chisel consists of 4 Scala projects; each is its own separate compilation unit:
-
-* [`core`](core) is the bulk of the source code of Chisel, depends on `macros`
+Chisel consists of several Scala projects; each is its own separate compilation unit:
```

Comment:
Nit: why are we putting these arguably less interesting things first in the list? Should we go in at least alphabetical order?

Reply:
It's contributor documentation so it's helpful to know where code is. Alphabetical order makes sense although things that come first in the alphabet depend on things that come later.